internal/command/flags.go

```diff
@@ -51,3 +52,7 @@ func installServingOption(targetCmd *cobra.Command) {
 func installEventingOption(targetCmd *cobra.Command) {
 	targetCmd.Flags().BoolVar(&installEventing, "install-eventing", false, "install Eventing on quickstart cluster")
 }
+
+func installKindRegistryOption(targetCmd *cobra.Command) {
+	targetCmd.Flags().BoolVar(&installKindRegistry, "registry", false, "install registry for kind quickstart cluster")
```

Comment:
Maybe start with cap `Kind`?
```suggestion
	targetCmd.Flags().BoolVar(&installKindRegistry, "registry", false, "install registry for Kind quickstart cluster")
```